labs/computing-experiment-subjects/index.md

```diff
@@ -0,0 +1,213 @@
+# Computing Experiment Datasets #1: Experiment Subjects
+
+This Lab is part of a multi-part series focused on computing useful experiment datasets. In this Lab, we'll use [PySpark](https://spark.apache.org/docs/latest/api/python/index.html) to compute _experiment subjects_ from [Optimizely Enriched Event](https://docs.developers.optimizely.com/optimizely-data/docs/enriched-events-export) ["Decision"](https://docs.developers.optimizely.com/optimizely-data/docs/enriched-events-data-specification#decisions-2) data.
+
+<!-- We use an external image URL rather than a relative path so that this notebook will be rendered correctly on the Optimizely Labs website -->
+![Experiment subjects computation](https://raw.githubusercontent.com/optimizely/labs/master/labs/computing-experiment-subjects/img/subjects_computation.png)
+
+**Experiment subjects** are the individual units that are exposed to a control or treatment in the course of an online experiment.  In most online experiments, subjects are website visitors or app users. However, depending on your experiment design, treatments may also be applied to individual user sessions, service requests, search queries, etc. 
+
+## How to run this notebook
+
+This notebook lives in the [Optimizely Labs](http://github.com/optimizely/labs) repository.  You can download it and everything you need to run it by doing one of the following
+1. Downloading a zipped copy of this Lab directory on the [Optimizely Labs page](https://www.optimizely.com/labs/computing-experiment-subjects/)
+2. Downloading a [zipped copy of the Optimizely Labs repository](https://github.com/optimizely/labs/archive/master.zip) from Github
+3. Cloning the [Github respository](http://github.com/optimizely/labs)
+
+Once you've downloaded this Lab directory (on its own, or as part of the [Optimizely Labs](http://github.com/optimizely/labs) repository, follow the instructions in the `README.md` file for this Lab.
+
+If you wish to run this notebook using a custom data directory, set the `OPTIMIZELY_DATA_DIR` environment variable to point to the 
+
+## Analysis parameters
+
+We'll use the following global variables to parameterize our computation:
```

Comment:
I think this kind of instruction makes sense when inside the environment of a jupyter lab, but it may seem odd to see it in a webpage. For example, when I see "We'll use the following...", I think, where do I put this code? Do I need to put it in a file? What filename? How do I run it?

Obviously this is clearer when someone is running the notebook. So I'm wondering if this content should _only_ live in the notebook so that it makes the most sense in the context its designed for.

Reply:
Hm, I get the point, but I wouldn't want to omit them from the page since they are referenced later on in the notebook.  I could add a note this page being a notebook that can actually be executed (see instructions below)

Reply:
Added a note to the intro.